manifests/v/VSDC/Editor/7.1.1.390/VSDC.Editor.installer.yaml

```
@@ -1,17 +1,17 @@
-# Created with YamlCreate.ps1 v2.1.0 $debug=NVS1.5-1-19041-1320
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
-
-PackageIdentifier: VSDC.Editor
-PackageVersion: 6.9.5.382
-MinimumOSVersion: 10.0.0.0
-InstallerType: inno
-UpgradeBehavior: install
-Installers:
-- Architecture: x64
-  InstallerUrl: https://downloads.videosoftdev.com/video_tools/video_editor_x64.exe
-  InstallerSha256: CB109D602889CF1F9E18659D46D9E4D2E00DF95FFA8F502B61936453E22A0E6A
-- Architecture: x86
-  InstallerUrl: https://downloads.videosoftdev.com/video_tools/video_editor_x32.exe
-  InstallerSha256: 15C53C36CED9DEB3D93B2AABD740C2F19FF5D081BACED2FE63B232028EBDE2E7
-ManifestType: installer
-ManifestVersion: 1.1.0
+# Created with YamlCreate.ps1 v2.1.2 $debug=AUSU.5-1-19041-1645
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: VSDC.Editor
+PackageVersion: 7.1.1.390

```

```suggestion
PackageVersion: 7.1.1.392
```
Version got updated.